noir-projects/noir-contracts/contracts/token_contract/src/types/token_note.nr

```diff
@@ -48,14 +48,8 @@ impl NoteInterface<TOKEN_NOTE_LEN, TOKEN_NOTE_BYTES_LEN> for TokenNote {
     }
 
     fn compute_note_hiding_point(self) -> Point {
-        assert(self.header.storage_slot != 0, "Storage slot must be set before computing note hiding point");
-
-        // TODO(#7772): decompose amount with from_field_unsafe or constrain it fits into 1 limb
```

Comment:
Tackling this issue by using from_field_unsafe is fine because from_field_unsafe tries does the decomposing in an unconstrained context and hence does not add to gate counts.